sdk/core/core-client/review/core-client.api.md

```diff
@@ -292,19 +292,23 @@ export interface SequenceMapperType {
 // @public
 export interface Serializer {
     // (undocumented)
-    deserialize(mapper: Mapper, responseBody: any, objectName: string): any;
+    deserialize(mapper: Mapper, responseBody: any, objectName: string, options?: SerializerOptions): any;
     // (undocumented)
     readonly isXML: boolean;
     // (undocumented)
     readonly modelMappers: {
         [key: string]: any;
     };
     // (undocumented)
-    serialize(mapper: Mapper, object: any, objectName?: string): any;
+    serialize(mapper: Mapper, object: any, objectName?: string, options?: SerializerOptions): any;
     // (undocumented)
     validateConstraints(mapper: Mapper, value: any, objectName: string): void;
 }
 
+// @public
+export interface SerializerOptions extends XmlOptions {
```

Comment:
is it worth eagerly creating this interface if it's empty for now? Should we just reuse `XmlOptions` directly until the Serializer gets a custom option?

Alternatively, should we nest this like:

```ts
interface SerializerOptions {
   xml: XmlOptions;
}
```

So that options specific to a format are split apart? This might help if we ever want to separate XML and JSON serialization.

Reply:
> is it worth eagerly creating this interface if it's empty for now? Should we just reuse `XmlOptions` directly until the Serializer gets a custom option?

I was trying to avoid future breaking changes and it feels right to have xml options for core-xml, and serializer options in serailzer. 

> Alternatively, should we nest this like:

I debated between nesting and extending too. Yes nesting sounds better. I will do that.